### PR TITLE
Remove issue endpoints

### PR DESCRIPTION
**Pull request by scholvat**

### CODE REVIEW

**Comment by scholvat** [comment]
Full disclosure, I didn't do any testing on this code since I don't have the dev environment setup.

File: controllers/serverController.js [hunk 1]
```diff
@@ -58,13 +59,15 @@ router.get('/:id', (req, res) => {
   });
 });
 
+/* TODO: re-add delete endpoint once security becomes a thing
 router.put('/:id', checkAccessKey, (req, res) => {
```
Comment (picklechips) [inline controllers/serverController.js:63]: isn't this what `checkAccessKey` is for?

File: controllers/userController.js [hunk 2]
```diff
@@ -71,12 +71,13 @@ router.post('/login', (req, res) => {
   });
 });
 
+/* TODO: re-add delete endpoint once security becomes a thing
 router.delete('/:id', (req, res) => {
```
Comment (picklechips) [inline controllers/userController.js:75]: Can we not just add `checkAccessKey` to the middleware

**Review by picklechips — COMMENTED** [review]
I could be missing something, but I think checkAccessKey is all we need to ensure these calls are secure. It checks that the caller is the corresponding server.

I believe `post /` is the only problematic scenario, which just adds a new server

File: controllers/serverController.js (hunk 1 above)
Comment (scholvat) [reply controllers/serverController.js:63]: The access key is a public variable that is exposed, https://skybunk-auth-production.herokuapp.com/users. It offers no security other than obscurity.

Comment (picklechips) [reply controllers/serverController.js:63]: These are salted/hashed, in the same way user passwords are. We probably shouldn't be returning it in the response but from a security perspective there's nothing you can do with that value without knowing the salt & the original value

https://crackstation.net/hashing-security.htm

Comment (mipollex) [reply controllers/serverController.js:63]: I agree. The access key shouldn't be recoverable from the salted hash. However we still shouldn't be returning it in the response.
Realistically nothing is using these endpoints yet, so either commenting or adding the `checkAccessKey` would work. My vote is for the `checkAccessKey` path.

Comment (scholvat) [reply controllers/serverController.js:63]: True, I didn't realize it was salted hash. We can add checkAccessKey to everything, and then remove the post endpoint for now (I don't know the best way to secure the post endpoint)

**Pull request closed without merging**